### PR TITLE
34 get postpost idlikes

### PR DESCRIPTION
**Pull request by DLiSWE**
get post likes endpoint created

### CODE REVIEW

**Review by Matt-Tan15 — APPROVED** [review]
lgtm!

File: src/entities/sql/users.entity.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import { BaseEntity, Entity, PrimaryGeneratedColumn, Column, Unique, CreateDateColumn, UpdateDateColumn, OneToMany } from 'typeorm';
-import { ConnectionsEntity } from '.';
+import { BaseEntity, Entity, PrimaryGeneratedColumn, Column, Unique, CreateDateColumn, UpdateDateColumn, OneToMany, ManyToMany } from 'typeorm';
```
Comment (raylu511) [inline src/entities/sql/users.entity.ts:1]: What are these imports for? `ManyToMany and PostEntity`

File: src/services/post.service.ts [hunk 2]
```diff
@@ -36,6 +36,16 @@ class PostService {
     if (posts.length < 1) throw new HttpException(404, 'There are currently no posts');
     return posts;
   }
+
+  public async getLikesFromPost(id: number): Promise<UsersLikedPostDto[]> {
+    const getLikes = await PostEntity.find({
+      relations: ['likes'],
+      where: { id: id },
```
Comment (raylu511) [inline src/services/post.service.ts:43]: nit: you can just do ` where: { id },
`

File: src/services/post.service.ts [hunk 3]
```diff
@@ -36,6 +36,16 @@ class PostService {
...
+      select: { likes: { firstName: true, lastName: true, profilePic: true } },
+    });
+    if (getLikes.length === 0) throw new HttpException(404, '0 likes');
```
Comment (raylu511) [inline src/services/post.service.ts:46]: I think we can remove this check and just return an empty array

**Review by emir4ik-code — APPROVED** [review]

**Review by raylu511 — APPROVED** [review]

**Pull request merged**